playbooks/installation/installation.adoc

```diff
@@ -1,280 +1,108 @@
+  ** 2 cores
+  ** 4 GB Memory
+  ** 10 GB Root Drive
 
-==== Questions to Ask
+NOTE: We chose the resource sizing above based on some reasonable starting sizes for hosts based on our experiences in the field. Before building further environments, it is important to go through a real production sizing exercise. For more information on how we recommend going about this, see our link:/playbooks/operationalizing/environment_sizing{outfilesuffix}[Environment Sizing Guide].
 
-. What is the total workload landscape?
-. What is the CPU and memory footprint of to-be-migrated workloads?
-. What is the platform type (bare metal, virtualized, IaaS)? (Failure is more difficult to handle in bare metal circumstances and may require more nodes)
-. Can all workloads run on the same nodes or do you require segregation?
+NOTE: We recommend segregating local disks this way in order to provide more stability to the separate components that will write to these locations. For more information about why we configure local storage this way, read link:/playbooks/operationalizing/local_storage{outfilesuffix}[Considerations for Configuring Local Storage on OpenShift Hosts]
 
-=== Storage Considerations
+Additionally, the following is assumed about the environment OpenShift is being installed into:
 
-Without the proper protections in place (through either Disk partitioning or volume groups), we leave ourselves at risk that a component or service running on the host OS could potentially fill the disk, preventing either of these from writing to disk and corrupting the OSE install.
+* An external load balancer is available for use to load balance to the *Master* and *Infrastructure Node* servers
+* An external DNS infrastructure is available for creating hostnames for the servers, as well as a wildcard entry for application traffic routing
+* An external storage infrastructure capable of providing NFS volumes
 
-==== OpenShift Storage Requirements
+NOTE: For more information, see the link:https://docs.openshift.com/enterprise/latest/install_config/install/prerequisites.html[Official Documentation regarding OpenShift Installation Prerequisites].
 
-There are two main components of OpenShift that require specialized storage considerations to ensure environment stability. These components are the OpenShift DataStore (etcd) and the Container Platform (Docker).
+=== Configure DNS
 
-===== DataStore (etcd)
+OpenShift expects properly configured DNS in order to work properly. The assumption of this document is that you will be using some external DNS system (Corporate DNS) to do this. If you don't have en existing DNS system or don't with to use it for the purpose of this setup, then you will have to create one. We recommend creating a bind server if this will be a permanent setup, or using DNSMasq as a temporary workaround if you plan to ultimately tie into a Corporate DNS in the future.
```

Comment:
updated wording in 3rd sentence:

If you don't have en existing DNS system or do not plan to use it for the purpose of this setup, then you will have to create one.